connexion/spec.py

```diff
@@ -24,33 +25,39 @@
 validate_properties = Draft4Validator.VALIDATORS["properties"]
 
 
-def validate_defaults(validator, properties, instance, schema):
-    """Validate `properties` subschema.
+def create_validate_default_fn(instance_validator: Draft4Validator) -> t.Callable:
+    """Creates a validation function for property defaults. This validation function will be used
+    by a validator that validates an openapi spec against the openapi schema. The default value
+    however needs to be validated against the openapi spec itself.
+
+    Args:
```

Comment:
Don't think the Google docstring format is used elsewhere. To keep it consistent, let's use [the Sphinx docstring](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format) format here as well?

Reply:
Good catch. Wrong project defaults on new laptop :)